modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase

```diff
@@ -1221,6 +1221,48 @@ definition {
 			status = "Scheduled");
 	}
 
+	@description = "This is a test for bug COMMERCE-11209. As the Admin I can delete a product image when the image is set via headless API."
+	@priority = 5
+	test CanDeleteProductImageWhenImageIsUploadedViaAPI {
+		task ("Given a Minium Site is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		}
+
+		task ("And for the product ABS Sensor is uploaded an image using API") {
+			var responseBody = CommerceJSONProductsAPI._patchCommerceProductImageUsingERC(
+				cdnEnabled = "true",
+				cdnURL = "http://mirrors.lax.liferay.com/github.com/liferay/liferay-portal/portal-web/test/functional/com/liferay/portalweb/dependencies/Commerce_Black.jpg",
+				imageTitle = "title2",
+				productERC = "MIN93015minium-initializer");
```

Comment:
If `responseBody` is not used anywhere in the test, it is not necessary to have the responseBody returned